tutorials/plot_sensors_time_frequency.py

```diff
@@ -115,6 +115,15 @@
 plt.show()
 
 ###############################################################################
+# You can also create a joint plot that contains both the aggregated TFR
+# across channels and topomaps at specific times and frequencies.
+
+power.plot_joint(picks=range(82, 84), baseline=(-0.5, 0), mode='logratio',
+                 title='auto', timefreqs=[(1, 10), (1, 19)],
+                 timefreqs_average=[(1, 4), (1, 6)])
```

Comment:
No, I mean:

```
timefreqs=[(1, 10), (1, 19)],
timefreqs_average={(1, 10):(1, 4), (1, 19):(1, 6)}
```

Could also just go
`timefreqs=[(1, 10), (1, 19)]` (no averaging)
`timefreqs_average={(1, 10):(1, 4), (1, 19):(1, 6)}` (averaging by dict keys)

@dengemann what do you think?